app/src/main/java/ml/docilealligator/infinityforreddit/adapters/CommentsRecyclerViewAdapter.java

```diff
@@ -981,6 +977,20 @@ public void editComment(String commentAuthor, boolean isSubmitter, String commen
         }
     }
 
+    public void editComment(Comment fetchedComment, Comment originalComment, int position) {
+        if (!mVisibleComments.get(position).equals(originalComment)) {
+            position = mVisibleComments.indexOf(originalComment);
```

Comment:
`indexOf` can return `-1` and crash the app

Reply:
Actually even the first `mVisibleComments.get(position)` can throw if you request removed comment near the end of the list, then collapse some comments before it loads